packages/neuron-wallet/src/services/networks.ts

```diff
@@ -66,6 +66,8 @@ enum NetworksKey {
 }
 
 const oldDefaultNames = ['Default', 'default node', presetNetworks.networks[0].name]
+// Before 0.106.0 version the default remote's value is http://localhost:8114.
```

Comment:
```suggestion
// Before 0.106.0 version the default remote's value is http://localhost:8114.
// "localhost" was deprecated because of https://github.com/Magickbase/neuron-public-issues/issues/122
```